tests/Controller/AddressBook/CreateAddressApiTest.php

```diff
@@ -135,7 +142,13 @@ public function it_does_not_allow_user_to_add_new_address_to_address_book_withou
 JSON;
 
         $response = $this->createAddress($data);
-        $this->assertResponse($response, 'address_book/validation_create_address_book_with_wrong_province_response', Response::HTTP_BAD_REQUEST);
+        if (Kernel::VERSION_ID < 60000) {
+            $this->assertResponse($response, 'address_book/SF5/validation_create_address_book_with_wrong_province_response', Response::HTTP_BAD_REQUEST);
+        }
+
+        if (60000 <= Kernel::VERSION_ID) {
+            $this->assertResponse($response, 'address_book/SF6/validation_create_address_book_with_wrong_province_response', Response::HTTP_BAD_REQUEST);
+        }
```

Comment:
```suggestion
        if (Kernel::VERSION_ID < 60000) {
            $this->assertResponse($response, 'address_book/SF5/validation_create_address_book_with_wrong_province_response', Response::HTTP_BAD_REQUEST);
        } else {
            $this->assertResponse($response, 'address_book/SF6/validation_create_address_book_with_wrong_province_response', Response::HTTP_BAD_REQUEST);
        }
```

or with an early return.